testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -754,6 +755,31 @@ public void testShowCreateInformationSchemaTable()
                         ")");
     }
 
+    @Test
+    public void testRenameSchema()
+    {
+        if (!hasBehavior(SUPPORTS_RENAME_SCHEMA)) {
+            assertQueryFails(
+                    format("ALTER SCHEMA %s RENAME TO %s", getSession().getSchema(), getSession().getSchema() + randomTableSuffix()),
```

Comment:
Oooops, getSession.getSchema return Optional which becomes `Optional[default]` for example.
Maybe extract a string variable like `String schemaName = getSession().getSchema().orElseThrow()`

Reply:
lol